apps/microtvm/ethosu/README.md

```diff
@@ -43,6 +43,10 @@ If the demo is not run in the ci_cpu Docker container, then you will need the fo
 
 You will also need TVM which can either be:
   - Built from source (see [Install from Source](https://tvm.apache.org/docs/install/from_source.html))
+    - When building from source, the following need to be set in config.cmake:
```

Comment:
Do we also need `USE_MICRO_STANDALONE_RUNTIME`?

Reply:
No, `USE_MICRO` is sufficient for the demo.